InterviewQuestionsTests/Chapters/BitInsertions/BitInsertions.Tests.cs

```diff
@@ -0,0 +1,20 @@
+using InterviewQuestions.Chapters.BitInsertion;
+
+namespace InterviewQuestionsTests.Chapters.BitInsertion
+{
+    [TestClass] 
+    public class BitInsertionTest
+    {
+        [TestMethod]
+        public void BitInsertionMethod()
+        {
+            BitInsertions test1 = new BitInsertions();
+            int temp =test1.BitInsertionSolution(128,19,2,6);
+            int temp1 =test1.BitInsertionSolution(65,12,1,4);
+
+            Assert.AreEqual(temp,204);
+            Assert.AreEqual(temp1,89);
+        }
+    }
```

Comment:
Could you convert this test to a parametric test and use the following test cases?

[DataRow(128, 19, 2, 6, 204)]
[DataRow(65, 12, 1, 4, 89)]
[DataRow(2560, 7, 3, 5, 2616)]
[DataRow(145, 10, 2, 5, 169)]